### PR TITLE
Show indicator when repository is archived

### LINKED ISSUE #7183: Archived Repositories
This comment/issue is really talking about the GitHub Desktop app (I'm using Windows) and how it displays repositories when the "Clone repository..." menu item is used under the "File" menu with repositories marked on the github.com website as archived.

Is there a way to somehow mark a repository in the list of clone-able repositories as "Archived" so that when someone tries to clone a remote GITHUB repository in the desktop app that it's clear that the GITHUB archive in the list has been marked as "archived" on GITHUB?  I don't know how the "archived" flag works in GITHUB, so I don't know if there is a way to tell if a repository has been archived, but the GITHUB website shows an "archived" status when a repository is archived.

On the GITHUB website, archived repositories have a little box with the word archived next to the repository name 
image

This would be useful as either an option to show/hide archived repositories or maybe some kind of mark on the repository list to let users know that the repository was archived.  I still want to be able to clone a repository because I still want to have access to the code, but some sign that the repository has been archived would be helpful.

Thanks for listening.

### PR DESCRIPTION
**Pull request by niik**
Closes #7183

## Description

This adds an archived "badge" to archived repository in the clone dialog

### Screenshots

image

## Release notes

Notes: [Added] Show when a repository has been archived in the clone dialog

### CODE REVIEW

File: app/styles/ui/_cloneable-repository-filter-list.scss [hunk 1]
```diff
@@ -16,4 +16,20 @@
...
+  .clone-repository-list-item {
+    .name {
+      flex-grow: 1;
+    }
+
+    .archived {
+      flex-shrink: 0;
+      margin-left: var(--spacing-half);
+      font-size: var(--font-size-xs);
+      border: var(--base-border);
```
Comment (sergiou87) [inline app/styles/ui/_cloneable-repository-filter-list.scss]: The border color doesn't look good in dark mode IMO, I would expect a lighter color:

image

**Review by sergiou87 — CHANGES_REQUESTED** [review]
Works as expected, but the border has low contrast with the background in the dark theme

File: app/styles/ui/_cloneable-repository-filter-list.scss (hunk 1 above)
Comment (niik) [reply app/styles/ui/_cloneable-repository-filter-list.scss]: Good shout, I used the contrast border instead, looks much better IMO

image

image

**Review by sergiou87 — APPROVED** [review]
Looks perfect now, thanks for the quick fix! :shipit:

**Pull request merged**